### PR TITLE
fix: timeouts due to audio, video elements

### PR DESCRIPTION
**Pull request by mohanraj-r**
Related to [audio, video elements cause timeout in jest even with audio/video rules disabled · Issue #2528 · dequelabs/axe-core](https://github.com/dequelabs/axe-core/issues/2528)

### Fixes 🐛 
* **preset-rules:** disable preloading media in axe config to fix timeout issue (3cffead)

### Chores 🧹
- bump version of packages as they hadn't been bumped since [0.2.0-beta](https://github.com/salesforce/sa11y/releases/tag/v0.2.0-beta)
- add cmd to ensure pkg versions are bumped (85025d4)

### Documentation 📝 
- add caution notices about color contrast and headless mode (3f9f2dc)

### CODE REVIEW

File: packages/preset-rules/src/a11yConfig.ts
Comment (mohanraj-r) [inline packages/preset-rules/src/a11yConfig.ts:39]: @trevor-bliss @cordeliadillon This one line is essentially the fix for audio/video timeout issue. Rest of the changes are updating tests, docs and bumping versions.

Can you please review this when possible. I will get a release to npm with this fix once the PR is merged.

File: packages/jest/README.md
Comment (cordeliadillon) [inline packages/jest/README.md:77]: Grammatical typo -- remove "is"

File: packages/jest/README.md
Comment (cordeliadillon) [inline packages/jest/README.md:78]: Good advice here!

**Review by cordeliadillon — APPROVED** [review]

File: packages/jest/README.md
Comment (mohanraj-r) [reply packages/jest/README.md]: Good catch, thanks - fixed

File: packages/jest/README.md
Comment (mohanraj-r) [reply packages/jest/README.md:78]: Thanks!

File: .circleci/config.yml
Comment (trevor-bliss) [inline .circleci/config.yml:15]: I'm not sure about this. I don't really understand why we need `vertioner` to begin with though.

File: package.json
Comment (mohanraj-r) [reply package.json:32]: It is a small util to check if the `version` field in `package.json` file in current branch is different than master. I am using it to ensure the versions for packages that are changed are bumped without fail.

File: .circleci/config.yml
Comment (mohanraj-r) [reply .circleci/config.yml:15]: Yeah agree, this is not ideal. But CircleCI seems to map local master branch to the current PR or something. Have been thinking about a better solution - will keep looking.

File: package.json
Comment (trevor-bliss) [reply package.json:32]: So the expectation is every PR must bump the version of all packages?

Comment (mohanraj-r) [reply package.json:32]: Not really -  only the changed packages (`yarn lerna exec --since master ..` takes care of that) and the monorepo root version.

Comment (trevor-bliss) [reply package.json:32]: Ok interesting. I see a couple potential issues with this flow:

1. There can be version conflicts between PRs. Imagine there are several PRs open at the same time and they all bump the version to the next patch version. As soon as one gets merged, all other PRs must go in and bump their version again. This potentially repeats for every open PR when a different PR is merged. 

2. There may be versions in github without corresponding NPM releases. Unless we release every commit to master to NPM, there may be several version bumps before a new version in NPM. Not necessarily a bad thing but can be confusing IMO.

3. We put the burden of versioning on the PR author. When I contribute to other projects I just focus on the code fix rather than worrying about whether something should be a patch vs. minor release. 

Just my opinion but I think it makes more sense to leave the versions untouched in PRs and then do a release from `master` or a release branch and we as project maintainers decide when that release happens and what version to bump to.

**Review by trevor-bliss — DISMISSED** [review]
Have some opinions on the versioning and release flow as commented, but the actual code changes here look good.

File: package.json
Comment (mohanraj-r) [reply package.json:32]: Thanks @trevor-bliss - Those are very valid concerns 👍 
I have created a issue to track this and continue discussion https://github.com/salesforce/sa11y/issues/40

Comment (mohanraj-r) [reply package.json]: @trevor-bliss I have removed the version check on mono-repo root due to [a potential issue of it failing after merging into master](https://github.com/hectotruj/vertioner/issues/5)

I was also able to remove force setting the master branch in CI branch since `lerna exec` seems to work fine without it. 

Can you please give a glance and approve. 
I will release to npm and work on improving versioning as part of #40

File: .circleci/config.yml
Comment (mohanraj-r) [reply .circleci/config.yml]: This has been removed now.

**Review by trevor-bliss — APPROVED** [review]
:shipit:

**Pull request merged**
